test/trap.pointer-events.test.js

```diff
@@ -20,6 +46,7 @@ jest.mock('../src/constants', () => {
 describe('browser with pointer events', () => {
   beforeAll(() => {
     document.body.innerHTML = initialHtml;
+    window.PointerEvent = PointerEventFake;
```

Comment:
Is this change permanent for the rest of the tests?  Or does the test framework reinitialize the `window` object all the time?

Reply:
The window object gets reinitialized in the other suites (and so PointerEvent will be undefined) as described in these issues:  
https://github.com/testing-library/dom-testing-library/issues/558 and
https://github.com/jsdom/jsdom/pull/2666

Reply:
Thx!